ui/nuxeo-data-table/data-table-settings.js

```diff
@@ -53,43 +54,43 @@ import '../widgets/nuxeo-dialog.js';
 
           .buttons {
             @apply --buttons-bar;
+            margin-top: 0px;
           }
         </style>
 
         <nuxeo-connection id="nxcon"></nuxeo-connection>
 
         <nuxeo-dialog id="columnsSettingsPopup" with-backdrop on-iron-overlay-closed="_onSettingsClosed">
           <h2>[[i18n('tableSettings.columnSettings')]]</h2>
-          <div class="paper-content layout horizontal">
-            <div class="layout vertical">
-              <div class="row layout horizontal">
-                <div class="label-container">
-                  <label></label>
+          <paper-dialog-scrollable>
+            <div class="paper-content layout horizontal">
+              <div class="layout vertical">
+                <div class="row layout horizontal">
+                  <dom-repeat items="[[columns]]" as="column">
+                    <template>
+                      <tr>
+                        <td>
+                          <paper-checkbox noink checked="{{!column.hidden}}"></paper-checkbox>
+                        </td>
+                        <td>
+                          [[column.name]]
+                        </td>
+                      </tr>
+                    </template>
+                  </dom-repeat>
+                  <table></table>
                 </div>
-                <dom-repeat items="[[columns]]" as="column">
-                  <template>
-                    <tr>
-                      <td>
-                        <paper-checkbox noink checked="{{!column.hidden}}"></paper-checkbox>
-                      </td>
-                      <td>
-                        [[column.name]]
-                      </td>
-                    </tr>
-                  </template>
-                </dom-repeat>
-                <table></table>
               </div>
             </div>
-          </div>
+          </paper-dialog-scrollable>
           <div class="buttons horizontal end-justified layout">
             <div class="flex start-justified">
               <paper-button noink on-click="_resetSettings"
                 >[[i18n('tableSettings.columnSettings.reset')]]</paper-button
               >
             </div>
             <paper-button noink class="primary" dialog-dismiss
-              >[[i18n('tableSettings.columnSettings.close')]]</paper-button
+              >[[i18n('tableSettings.columnSettings.save')]]</paper-button
```

Comment:
Why turning `Close` into `Save`. Actually it is not saving anything because changes are persisted in the localstorage as soon as you check a checkbox. So `Close` was more accurate than `Save` IMO

Reply:
This was actually something I discussed with our designer. 

Right now, it's not clear that if you close the window, the modifications you did will persist or not. That's why we are suggesting the idea of "saving" the configuration. For the end user it doesn't matter what is happening under the hood: we just want to ensure him that whatever he did, will be applied.

Reply:
Could be `Save` or `Set`, I would say.